src/scheme/scheme.go

```diff
@@ -18,6 +18,9 @@ package scheme
 
 import (
 	dynatracev1alpha1 "github.com/Dynatrace/dynatrace-operator/src/api/v1alpha1"
+	// needed to ensure proper package initialization order
+	_ "github.com/Dynatrace/dynatrace-operator/src/api/v1alpha1/dynakube"
+	_ "github.com/Dynatrace/dynatrace-operator/src/api/v1alpha1/edgeconnect"
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1/dynakube"
 	istiov1alpha3 "istio.io/client-go/pkg/apis/networking/v1alpha3"
 	corev1 "k8s.io/api/core/v1"
```

Comment:
not your change that has the problem here but
this package is a bit messed up, because the `v1beta1` package is a bit out of line

did a quick fix [here](https://github.com/Dynatrace/dynatrace-operator/commit/d7646960ea1e9d9e6f727258e2be0ab2d7340f29)

if you want to add it, then do it
otherwise I can create a follow up

Reply:
I've cherry-picked your commit.